client/App.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Remove this comment. If there is an unused variable, evaluate whether or not we need it and then update the code accordingly.

Reply:
I understand, it was an oversight.